enterprise-suite/scripts/lbc.py

```diff
@@ -464,6 +464,21 @@ def install(creds_file):
 
     helm_args = ''
     if len(args.helm) > 0:
+        # namespace can be passed to helm directly after -- and can take format -namespace <value> or --namespace=<value>
+        # remove namespace from helm args and assign it args.namespace
+        # handle --namespace <val>
+        if  "--namespace" in args.helm and args.helm.index("--namespace") < len(args.helm):
```

Comment:
Is the `args.helm.index` necessary here? Not sure why it needs that check if `"--namespace" in args.helm` succeeds.

Reply:
This check is to make sure there at least one more variable provided as value for namespace